CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Breaking changes
 
+- [#1522: Create govuk-branded.html template](https://github.com/alphagov/govuk-prototype-kit/pull/1522)
```

Comment:
We should mention that users now have to import macros before using them.

Reply:
Added, thanks.